control_msgs/msg/DynamicInterfaceValues.msg

```diff
@@ -0,0 +1,2 @@
+SingleInterfaceValue[] state_interface_values
+SingleInterfaceValue[] command_interface_values
```

Comment:
```suggestion
std_msgs/Header header

SingleInterfaceValue[] state_interface_values
SingleInterfaceValue[] command_interface_values
```

Reply:
```suggestion
std_msgs/Header header

InterfaceValue state_interface
InterfaceValue command_interface
```

Having values in the names seems redundant
I think with simple `InterfaceValue` message we could achieve the same

Reply:
already changed this but forgot to push, sry about that